packages/wormhole-sdk/src/bridge.ts

```diff
@@ -42,40 +49,71 @@ export class WormholeBridge {
   settings: AllBridgeSettings
 
   constructor({ srcDomain, dstDomain, srcDomainProvider, dstDomainProvider, settings }: WormholeBridgeOpts) {
-    this.srcDomain = srcDomain
+    this.srcDomain = getLikelyDomainId(srcDomain)
     this.dstDomain = dstDomain || getDefaultDstDomain(srcDomain)
     this.srcDomainProvider = srcDomainProvider || new ethers.providers.JsonRpcProvider(DEFAULT_RPC_URLS[this.srcDomain])
     this.dstDomainProvider = dstDomainProvider || new ethers.providers.JsonRpcProvider(DEFAULT_RPC_URLS[this.dstDomain])
 
     this.settings = { useFakeArbitrumOutbox: false, ...settings }
   }
 
+  private async _optionallySendTx(
```

Comment:
suggestion: if it doesn't use any of the class state make it a function at the bottom of the file instead